app/helpers/application_helper.rb

```diff
@@ -45,4 +45,11 @@ def table_of_contents_separator(options = {})
     contents = contents.join('<br />').html_safe # rubocop:disable Rails/OutputSafety
     render partial: 'catalog/table_of_contents', locals: { contents: contents }
   end
+
+  def manuscript_link(options = {})
+    return if options[:value].blank?
+    title = options[:document]['title_full_display']
+    title = title.partition(':')[2] if title.include?(':')
```

Comment:
Does this mean if their isn't a `:` then the text will == the "Page detail title" ?

Reply:
@mejackreed that's correct. The proper `p. 3: Manuscript title` format requires the manifest to have a `MODS/label`.

Would manuscript number would be a better fallback behavior once #858 is done?

Reply:
Here's an example, @mejackreed:

## bg021sq9590
![manuscript title](https://user-images.githubusercontent.com/5402927/32633360-0f091da6-c55c-11e7-8379-652bc75e49fa.png)

## fh878gz0315 (fallback behavior)
![manuscript number](https://user-images.githubusercontent.com/5402927/32633361-0f1c9df4-c55c-11e7-80bb-ffdd45865d89.png)